app/Phragile/MergedIntoTransaction.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Phragile;
+
+class MergedIntoTransaction extends Transaction {
+
+	const TYPE = 'mergedInto';
+
+	public function __construct($timestamp)
+	{
+		$this->timestamp = $timestamp;
+	}
+
+	public function getData()
```

Comment:
You're absolutely right. I've also first forgotten about it, and only realized that when replacing old code and spotted this "mergedinto". I'll add an explanation. Plus I think it would even make sense to rename this class to something like `MergeAndCloseTransaction` or `CloseByMergingTransaction`. Both names are not very appealing but maybe I'll manage to come up with something better while going through a rebase hell :)